draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -323,17 +323,17 @@
 			configurable on a per-node basis and MAY be
 			configurable on a per-interface basis.
 		</t>
-		<section anchor="LSPReceptionWindow" title="LSP Burst Window sub-TLV">
-			<t>The LSP Burst Window sub-TLV advertises the maximum number of LSPs that the node can receive without an intervening delay between LSP transmissions.</t>
+		<section anchor="LSPReceptionWindow" title="LSP Burst Size sub-TLV">
+			<t>The LSP Burst Size sub-TLV advertises the maximum number of LSPs that the node can receive without an intervening delay between LSP transmissions.</t>
 			<t>Type: 1</t>
 			<t>Length: 4 octets</t>
-			<t>Value: number of LSPs that can be sent back-to-back.</t>
+			<t>Value: number of LSPs that can be received back-to-back.</t>
 		</section>
 		<section anchor="InterfaceLSPTransmissionInterval" title="LSP Transmission Interval sub-TLV">
-			<t>The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be received on this interface, after the maximum number of un-acknowledged LSPs have been sent.</t>
+			<t>The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be received on this interface, following the reception of "LSP Burst Size".</t>

```

Comment:
Caveat: In areas (like this one) where the text is most relevant to the receiver driven algorithm, I defer to the proponents of that algorithm. Please consider that when responding to my comments.

The problem that I have here has nothing to do with the proposed change - my comment applies equally to the old/new text.

Associating this value with applying "after" or "following" a burst of LSPs seems inappropiate and somewhat inaccurate. As stated elsewhere in the document, this is really the steady state reception rate which the receiver can sustain. If a burst occurs, I would think the sender is obligated to delay sending additional LSPs on that interface long enough so that the average LSP transmission rate over some "reasonable interval" (e.g., a second) matches the advertised LSP Transmission Interval. 

As written/revised, the text suggests suggest that the sender MUST send a burst first before conforming to the steady state rate.

Reply:
Thanks for the feedback and the caveat. Regarding the caveat, I would note that multiple existing implementations already send a burst for the N first LSPs. Knowing the right value "N" acceptable for the receiver would be relevant IMO.

Regarding the text, I agree with you but I'm not sure how to address your point while minimizing the changes.
May be
OLD: The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be received on this interface, following the reception of "LSP Burst Size".
NEW: The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be sustained on this receiving interface.

And may be for the Burst Size
OLD: The LSP Burst Size sub-TLV advertises the maximum number of LSPs that the node can receive without an intervening delay between LSP transmissions.
NEW: The LSP Burst Size sub-TLV advertises the maximum number of initial LSPs that the node can receive without an intervening delay between LSP transmissions.

OLD: Value: number of LSPs that can be received back-to-back.
NEW: Value: number of initial LSPs that can be received back-to-back.

Does this help? Comment and suggestions welcomed.

Reply:
Also agree with Les that there should be no ordering between the burst size and the sustained rate.

On your proposed change Bruno, I would not mention "initial LSPs" but just "LSPs" for the Burst size description. I think we want to allow an implementation that uses a timer controlled by the LSP Burst Size parameter to still send burst of packets when a timer fires, to reduce the rate of such timers.

Reply:
Thank you Guillaume. Very good point.